stackstate_checks_base/stackstate_checks/base/data/agent_requirements.in

```diff
@@ -1,3 +1,4 @@
+cm-client==33.0.0
```

Comment:
do we need this one ?

Reply:
Without that ` checksdev validate dep` will fail in gitlab pipeline job `test_stackstate_checks_base`